### ISSUE TITLE
Some Create Cloud Credential forms will "flash" fields when type in description box

### ISSUE THREAD

**Issue by davidnuzik** [rancher/dashboard#3866]
**Summary:**

This is an interesting behavior, if you create a cloud credential, such as "Create Amazon" or "Create S3-Compatible" and type in the description box, the other fields besides Name and Description will disappear until you finish typing. If you type slow enough they will flash on the screen.

**Environment:**
Rancher version: v2.6-head `f81199694` 8/16/21 9:08 am Pacific
Rancher cluster type: single-node docker
Docker version: 19.03
 
Downstream cluster type: n/a
Downstream K8s version: n/a

**Steps to Reproduce:**
1. Create a cloud credential from the "Cloud Credentials" page (not during cluster creation). Pick Amazon or S3-Compatible.
2. Start typing in the description field, as you type the other fields disappear but come back once you pause a a moment.
 
**Expected Result:**
The fields should not "flash" on the screen as you type or disappear while typing.

**Actual Result:**
The fields "flash" on screen if you type slow enough or just disappear when typing fast enough in the description field.

**Additional Info:**
![image](https://user-images.githubusercontent.com/45179589/129596736-8c7841f0-cb99-412c-adfa-5afabd4612cc.png)
Example with Amazon, if you type fast in the Description field, the Access Key, Secret Key, and Default Region drop down will disappear.

**Comment by davidnuzik** [comment]
I don't think this is something we have to fix for 2.6.0 in my opinion, it's just a mildly annoying visual bug.
I put in 2.6 milestone as this is correct QA procedure when filing new issues.

**Comment by nwmac** [comment]
@neillsom This may be because the form is being recreated/rendered each time the model changes
